### PR TITLE
bt-703: connect employer onboarding to api

### LINKED ISSUE #703: Connect file uploading to employer onboarding

### LINKED ISSUE #715: Connect uploading photo and company logo
When talent/employer adds a profile photo/company logo it should be displayed on the Preview page and Profile page

### PR DESCRIPTION
**Pull request by JosuerBague**
Closes #703

### CODE REVIEW

File: frontend/src/bundles/talent-onboarding/store/actions.ts [hunk 1]
```diff
@@ -23,13 +25,32 @@ const updateTalentDetails = createAsyncThunk<
...
+        files.push({
+            role: 'talentPhoto',
+            extension,
+            file: photo,
         });
+    }
 
-        restPayload.photoId = image.id;
-        restPayload.cvId = document.id;
+    if (files.length > 0) {
```
Comment (MaksymGavatuk) [inline frontend/src/bundles/talent-onboarding/store/actions.ts]: Magic number

File: frontend/src/bundles/employer-onboarding/store/actions.ts [hunk 2]
```diff
@@ -50,17 +52,50 @@ const saveEmployerDetails = createAsyncThunk<
...
+            if (companyLogo) {
+                const [extension] = companyLogo.name.split('.').reverse();
+                files.push({
+                    role: 'companyLogo',
+                    extension,
+                    file: companyLogo,
+                });
+            }
+
+            if (files.length > 0) {
```
Comment (MaksymGavatuk) [inline frontend/src/bundles/employer-onboarding/store/actions.ts]: The same

**Review by MaksymGavatuk — CHANGES_REQUESTED** [review]

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
I think we dont need package-lock.json here

**Review by nikita-remeslov — APPROVED** [review]

**Review by nika-sirak — CHANGES_REQUESTED** [review]
Failed to run localy. 

image

**Comment by JosuerBague** [comment]
> Failed to run localy.
> image

Fresh clone:
image

My old clone:
image


This is probably local to your npm @nika-sirak

**Comment by savinaDaria** [comment]
@JosuerBague I have almost the same issue as @nika-sirak 
It should be connected with package-lock.json which is included in your PR, please exclude any changes to it so it won't be applied locally.
I returned my previous package-lock and then it started to work, but it is bad solution, as there can be situations where you can't return to previous package-lock
image

**Comment by nikita-remeslov** [comment]
I have updated package-lock.json in the development.
Just make 
`git checkout origin/development^ package-lock.json`

**Review by nika-sirak — APPROVED** [review]

**Pull request merged**